### ISSUE TITLE
GDC Vault/Kaltura 403 Forbidden error on some videos

### ISSUE THREAD

**Issue by zorbathut** [ytdl-org/youtube-dl#21068]
## Checklist

- [X] I'm reporting a broken site support
- [X] I've verified that I'm running youtube-dl version **2019.05.11**
- [X] I've checked that all provided URLs are alive and playable in a browser
- [X] I've checked that all URLs and arguments with special characters are properly quoted or escaped
- [X] I've searched the bugtracker for similar issues including closed ones

## Verbose log

```
$ ./youtube-dl.exe https://gdcvault.com/play/1024340 -v
[debug] System config: []
[debug] User config: []
[debug] Custom config: []
[debug] Command-line args: ['https://gdcvault.com/play/1024340', '-v']
[debug] Encodings: locale cp1252, fs mbcs, out cp1252, pref cp1252
[debug] youtube-dl version 2019.05.11
[debug] Python version 3.4.4 (CPython) - Windows-10-10.0.17134
[debug] exe versions: none
[debug] Proxy map: {}
[GDCVault] 1024340: Downloading webpage
[Kaltura] 0_w0gn7oij: Downloading video info JSON
[Kaltura] 0_w0gn7oij: Downloading m3u8 information
[debug] Default format spec: best/bestvideo+bestaudio
[debug] Invoking downloader on 'http://cdnapi.kaltura.com/p/1670711/sp/167071100/playManifest/entryId/0_w0gn7oij/format/url/protocol/http/flavorId/0_5uiq4ysw?referrer=aHR0cHM6Ly9nZGN2YXVsdC5jb20='
ERROR: unable to download video data: HTTP Error 403: Forbidden
Traceback (most recent call last):
  File "C:\Users\dst\AppData\Roaming\Build archive\youtube-dl\ytdl-org\tmp4jj6sgve\build\youtube_dl\YoutubeDL.py", line 1915, in process_info
  File "C:\Users\dst\AppData\Roaming\Build archive\youtube-dl\ytdl-org\tmp4jj6sgve\build\youtube_dl\YoutubeDL.py", line 1854, in dl
  File "C:\Users\dst\AppData\Roaming\Build archive\youtube-dl\ytdl-org\tmp4jj6sgve\build\youtube_dl\downloader\common.py", line 364, in download
  File "C:\Users\dst\AppData\Roaming\Build archive\youtube-dl\ytdl-org\tmp4jj6sgve\build\youtube_dl\downloader\http.py", line 341, in real_download
  File "C:\Users\dst\AppData\Roaming\Build archive\youtube-dl\ytdl-org\tmp4jj6sgve\build\youtube_dl\downloader\http.py", line 109, in establish_connection
  File "C:\Users\dst\AppData\Roaming\Build archive\youtube-dl\ytdl-org\tmp4jj6sgve\build\youtube_dl\YoutubeDL.py", line 2227, in urlopen
  File "C:\Python\Python34\lib\urllib\request.py", line 470, in open
  File "C:\Python\Python34\lib\urllib\request.py", line 580, in http_response
  File "C:\Python\Python34\lib\urllib\request.py", line 502, in error
  File "C:\Python\Python34\lib\urllib\request.py", line 442, in _call_chain
  File "C:\Python\Python34\lib\urllib\request.py", line 685, in http_error_302
  File "C:\Python\Python34\lib\urllib\request.py", line 470, in open
  File "C:\Python\Python34\lib\urllib\request.py", line 580, in http_response
  File "C:\Python\Python34\lib\urllib\request.py", line 508, in error
  File "C:\Python\Python34\lib\urllib\request.py", line 442, in _call_chain
  File "C:\Python\Python34\lib\urllib\request.py", line 588, in http_error_default
urllib.error.HTTPError: HTTP Error 403: Forbidden
```

## Description

Some Kaltura-hosted GDC Vault videos don't properly download (example: https://gdcvault.com/play/1024340; can provide more URLs if helpful). This is not *all* Kaltura-hosted GDC Vault videos; https://gdcvault.com/play/1024344 works just fine.

I did some preliminary investigating of this myself. Both of these videos play in the web browser, and their embed tags are identical aside from a few IDs. If I take the URL that youtube-dl attempts to download, and put it in the browser, it doesn't work with 1024340 (403 forbidden, "an error occurred while processing your request") but does work with 1024344. I'm not sure what makes 1024340 different. The actual web browser playback seems to use an entirely different system, downloading a series of segment(number)_0_av.ts files, but this starts moving out of my extremely limited knowledge.

Some GDC Vault videos require a login to play, but the links I've provided do not.

Various GDC-Vault-related issues on Github recommend using --force-generic-extractor, but unfortunately this doesn't result in any change.

I've tested this on the current master git commit without any change.

Choosing a specific format *does* work - at least on this video, it's just the highest-quality one that breaks. It's possible this is really #3721 but I thought I'd report it in case there's something more immediate that can be done.

**Comment by pinkykitten** [comment]
I have the same issue.

**Comment by dhowe** [comment]
Same here

**Comment by remitamine** [comment]
fixed with 799756a3b3c794284ca52b9af482e1f03fc46833.
